src/devices/MemoryLCD/LSxxxB7DHxx.cs

```diff
@@ -0,0 +1,208 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Device.Gpio;
+using System.Device.Spi;
+using System.Threading;
+
+namespace Iot.Device.MemoryLcd
+{
+    /// <summary>
+    /// Memory LCD
+    /// </summary>
+    public abstract class LSxxxB7DHxx
```

Comment:
This should implement IDisposable to close the open pins and the Spi device connection.